application/src/test/java/uk/nhs/hee/tis/revalidation/changelog/SetDraftRecommendationsActualSubmissionDatesToNullChangeLogTest.java

```diff
@@ -0,0 +1,62 @@
+package uk.nhs.hee.tis.revalidation.changelog;
+
+import java.time.LocalDate;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.nhs.hee.tis.revalidation.entity.Recommendation;
+import uk.nhs.hee.tis.revalidation.repository.RecommendationRepository;
+
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.nhs.hee.tis.revalidation.entity.RecommendationStatus.READY_TO_REVIEW;
+
+@ExtendWith(MockitoExtension.class)
+class SetDraftRecommendationsActualSubmissionDatesToNullChangeLogTest {
+
+  @InjectMocks
+  SetDraftRecommendationsActualSubmissionDatesToNullChangeLog changeLog;
+
+  @Mock
+  RecommendationRepository recommendationRepository;
+
+  private static final String RECOMMENDATION_ID = "123";
+
+  private Recommendation draftWithDate = buildDraftWithActualSubmissionDate();
+  private Recommendation draftWithoutDate = buildDraftWithoutActualSubmissionDate();
```

Comment:
If these 2 objects are used in different tests, it would be better to set them as final.